src/pygama/math/histogram.py

```diff
@@ -125,23 +128,23 @@ def better_int_binning(x_lo=0, x_hi=None, dx=None, n_bins=None):
     # process inputs
     n_Nones = int(x_hi is None) + int(dx is None) + int(n_bins is None)
     if n_Nones > 1:
-        print('better_int_binning: must provide two of x_hi, dx or n_bins')
-        return
+        log.warning('better_int_binning: must provide two of x_hi, dx or n_bins')
+        raise Exception
```

Comment:
Some notes about errors/warnings:
* do not add the current function name, it is automatically added by logging / stacktrace
* Is it a warning? Then just call `log.warning()` and don't raise an exception
* Is it an error (e.g. execution cannot continue)? Then just raise an exception ([the one that makes most sense!](https://docs.python.org/3/library/exceptions.html#built-in-exceptions)) and pass the nessage to it.

You would change the above with:
```suggestion
        raise ValueError('must provide two of x_hi, dx or n_bins')
```
Maybe `ValueError` is not exactly the right one. In that case you would throw something more generic like `RuntimeError`.

Please update the rest of the PR accordingly.